src/unix-manager.c

```diff
@@ -1008,6 +1013,32 @@ TmEcode UnixManagerRegisterCommand(const char * keyword,
     SCReturnInt(TM_ECODE_OK);
 }
 
+/**
+ * \brief Set a timeout speed for the socket loop
+ *
+ * This function set a custom timeout value for the select loop
+ * around the UnixMain() function.
+ *
+ * \param seconds number of seconds for the timeout
+ * \param microseconds number of microseconds for the timeout
+ *
+ * \retval TM_ECODE_OK in case of success (always)
+ */
+TmEcode UnixManagerSetTimeout(int seconds, int microseconds)
```

Comment:
Since there isn't strong typing or named arguments, does it make more sense to just pass a timeval here?